README.md

```diff
@@ -1,117 +1,109 @@
 # OpenZeppelin Contracts for Cairo
 
-[![Tests and linter](https://github.com/OpenZeppelin/cairo-contracts/actions/workflows/coverage.yml/badge.svg)](https://github.com/OpenZeppelin/cairo-contracts/actions/workflows/coverage.yml)
-[![codecov](https://codecov.io/github/OpenZeppelin/cairo-contracts/branch/main/graph/badge.svg?token=LFSZH8RPOL)](https://codecov.io/github/OpenZeppelin/cairo-contracts)
+[![Lint and test](https://github.com/OpenZeppelin/cairo-contracts/actions/workflows/test.yml/badge.svg)](https://github.com/OpenZeppelin/cairo-contracts/actions/workflows/test.yml)
 
-**A library for secure smart contract development** written in Cairo for [StarkNet](https://starkware.co/product/starknet/), a decentralized ZK Rollup.
+**A library for secure smart contract development** written in Cairo for [Starknet](https://starkware.co/product/starknet/), a decentralized ZK Rollup.
 
-## Usage
-
-> ## ⚠️ WARNING! ⚠️
->
+> **Warning**
 > This repo contains highly experimental code.
-> Expect rapid iteration.
+> It has no code coverage checks.
+> It hasn't been audited.
 > **Use at your own risk.**
 
-### First time?
+## Usage
 
-Before installing Cairo on your machine, you need to install `gmp`:
+> **Warning**
+> Expect rapid iteration.
+> Some contracts or features are not ready to be deployed.
+> Check the **Unsupported** section below.
 
-```bash
-sudo apt install -y libgmp3-dev # linux
-brew install gmp # mac
-```
+### Prepare the environment
 
-> If you have any troubles installing gmp on your Apple M1 computer, [here’s a list of potential solutions](https://github.com/OpenZeppelin/nile/issues/22).
+1. [Install cairo](https://book.cairo-lang.org/ch01-01-installation.html)
+2. [Install scarb](https://docs.swmansion.com/scarb/download)
 
 ### Set up your project
 
-Create a directory for your project, then `cd` into it and create a Python virtual environment.
+Create a new project and `cd` into it.
```

Comment:
I wouldn't say I like adding the output in the commands when is not that helpful. 

```bash
$ scarb new my_project && cd myproject
```

This alternative also makes the copy button usable and is consistent with the solidity contracts README.

Reply:
well the `$` breaks the usability. i usually add the output when i think it's going to help the user know it ran successfully

Reply:
I agree with Eric in general on this; however, this is a brand new flow. I think there's value in users seeing the feedback they should receive at the expense of not being able to hit the `copy` button